sdk/core/azure-core-amqp/test/ut/management_tests.cpp

```diff
@@ -217,7 +217,7 @@ namespace Azure { namespace Core { namespace Amqp { namespace Tests {
       auto response = management.ExecuteOperation("Test", "Test", "Test", messageToSend);
       EXPECT_EQ(response.Status, ManagementOperationStatus::Ok);
       EXPECT_EQ(response.StatusCode, 200);
-      EXPECT_EQ(response.Description, "Successful");
+      EXPECT_EQ(response.Error.Description, "Successful");
```

Comment:
This is a bit odd. The error is populated, but the request was successful?

Reply:
That's also what HTTP does. Why is it odd?

Specifically, the management APIs return a tuple: HttpStatusCode, ErrorCondition, ErrorDescription. the Tuplie ErrorCOndition and ErrorDescription are packaged as an AmqpError, but an HttpStatusCode dosn't fit that model. So they are split out separately in the management operation response.